tests/statistics.cc

```diff
@@ -245,7 +246,7 @@ BOOST_AUTO_TEST_CASE(StatisticsTestAllNanButOne) { /* parasoft-suppress  LsstDm-
 
 }
 
-BOOST_AUTO_TEST_CASE(StatisticsTestImages) { /* parasoft-suppress  LsstDm-3-2a LsstDm-3-4a LsstDm-4-6 LsstDm-5-25 "Boost non-Std" */
+BOOST_AUTO_TEST_CASE(StatisticsTestImages, * utf::disabled()) { /* parasoft-suppress  LsstDm-3-2a LsstDm-3-4a LsstDm-4-6 LsstDm-5-25 "Boost non-Std" */
```

Comment:
This test is disabled by default because we should assume that afwdata is not setup (in case we drop afwdata from the stack in the future etc...). If it is setup (checked in the tests/Sconscript file), we pass a command line arg to the test - '--run_test=+StatisticsTestImages' to run this test.